RestaurantReservation.Db/DataModels/EmployeeView.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace RestaurantReservation.Db.DataModels
+{
+    public class EmployeeView
```

Comment:
the request in the exercise suggest adding more details about restaurant 
"Generate a query that retrieves employees with their respective restaurant details from a database view."